calico_versioned_docs/version-3.28/reference/installation/_crd-ref-docs/config.yaml

```diff
@@ -0,0 +1,69 @@
+processor:
+  # RE2 regular expressions describing types that should be excluded from the generated documentation.
+  ignoreTypes:
```

Comment:
@tmjd  I've copied verbatim from your list in main. I presume that this is entirely cumulative, and that adding names that don't exist in this version is okay. If that's a bad assumption, let me know.

Reply:
I would assume that your assumption is correct. I haven't tested it though so I can't say for certain.